src/components/Proposals/TargetAppBadge.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { AppBadge, GU } from '@aragon/ui'
+import { useDescribeVote } from '../../hooks/useDescribeVote'
+import { useOrgApps } from '../../providers/OrgApps'
+import { getAppPresentation } from '../../utils/app-utils'
+import LoadingSkeleton from '../Loading/LoadingSkeleton'
+
+function TargetApp({ vote }) {
+  const {
+    emptyScript,
+    targetApp,
+    loading: descriptionLoading,
```

Comment:
We can probably just go with `loading` in this new context, it's fairly obvious now what the loading refers to :)